### PR TITLE
docs are built in PR/CI and published as artifact if needed

### PR DESCRIPTION
**Pull request by spaceisfun**

### CODE REVIEW

**Review by hajya — COMMENTED** [review]
lgtm. :P

File: build/azure-pipelines.pr.yml
Comment (hajya) [inline build/azure-pipelines.pr.yml:73]: nice! i see this now.

File: build/azure-pipelines.pr.yml
Comment (hajya) [inline build/azure-pipelines.pr.yml:67]: consider just blasting it and replacing it. might be a little less error prone?

File: docs/README.md
Comment (hajya) [inline docs/README.md:26]: i think we can enforce this by just limiting/requiring specific sign off for check in against the gh-pages to core team as to avoid unneeded churn. 

that is, i'd like to have an explicit 'public docs' sign off..

**Review by hajya — COMMENTED** [review]
LGTM - Round 2 "the actually review it edition"

File: docs/README.md
Comment (spaceisfun) [reply docs/README.md:26]: I updated the gh-pages policies to require signoff before submit. Administrators (OneCore) must also follow this: https://github.com/microsoft/FactoryOrchestrator/settings/branch_protection_rules/18111886

File: build/azure-pipelines.pr.yml
Comment (spaceisfun) [reply build/azure-pipelines.pr.yml:67]: This seems reliable as is. I'm working on a separate PR to cleanup warnings which will also make the docs warning more obvious.

**Comment by spaceisfun** [comment]
improves #63

File: build/azure-pipelines.yml
Comment (hajya) [inline build/azure-pipelines.yml:77]: clever!!

File: docs/README.md
Comment (hajya) [inline docs/README.md:28]: NIT: 'live' documentation<-> code mismatches.. just so it's clear how it could be mismatched since folks might think the md files are the docs.

**Review by hajya — COMMENTED** [review]
lgtm. 1 NIT but really nbd.

**Review by hajya — APPROVED** [review]

**Pull request merged**
